lib/covermymeds_api/railtie.rb

```diff
@@ -0,0 +1,33 @@
+module CoverMyMeds
+  class Railtie < Rails::Railtie
+    config.covermymeds_api = ActiveSupport::OrderedOptions.new
+
+    config.covermymeds_api.default_host = "https://api.covermymeds.com/"
+
+    # Create (and cache) a configured API client instance using the id/secret
+    # stored in `Rails.application.secrets` and the configuration specified
+    # here and in `Rails.application.config.covermymeds_api`
+    def default_client
+      return @client if @client
+      @client = configured_client *credentials
```

Comment:
RIGHT. This was an artifact of a different approach that made it harder to use `||=`.